server/controllers/refer/referralsController.test.ts

```diff
@@ -145,4 +146,42 @@ describe('ReferralsController', () => {
       })
     })
   })
+
+  describe('checkAnswers', () => {
+    it('renders the referral check answers page', async () => {
+      const organisation = organisationFactory.build({ id: courseOffering.organisationId })
+      organisationService.getOrganisation.mockResolvedValue(organisation)
+
+      const person = personFactory.build()
+      personService.getPerson.mockResolvedValue(person)
+
+      const referral = referralFactory.build({ offeringId: courseOffering.id, prisonNumber: person.prisonNumber })
+      referralService.getReferral.mockResolvedValue(referral)
+
+      request.params.referralId = referral.id
+
+      const coursePresenter = createMock<CoursePresenter>({
+        audiences: courseAudienceFactory.buildList(1),
+        nameAndAlternateName: `${course.name} (${course.alternateName})`,
+      })
+      ;(CourseUtils.presentCourse as jest.Mock).mockReturnValue(coursePresenter)
+
+      const requestHandler = referralsController.checkAnswers()
+      await requestHandler(request, response, next)
+
+      expect(response.render).toHaveBeenCalledWith('referrals/checkAnswers', {
+        applicationSummaryListRows: ReferralUtils.applicationSummaryListRows(
+          courseOffering,
+          coursePresenter,
+          organisation,
+          person,
+        ),
+        courseOfferingId: courseOffering.id,
+        pageHeading: 'Check your answers',
+        person,
+        personSummaryListRows: PersonUtils.summaryListRows(person),
```

Comment:
We might be at the point that it would be useful to have a `PersonPresenter` rather than a public `personSummaryListRows` method (since we need to send both the person and their summary list here), like with `CoursePresenter` and `OrganisationWithOfferingEmailsPresenter`, but again one for another time